.github/workflows/steel-documentation.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy Steel Documentation
+
+on:
+  push:
+    branches: ["main"]
```

Comment:
Can we have this occur when we push a new `xX.Y` release tag? We'd like the GH pages to reflect the latest release, rather than `main`

Reply:
I updated the action to make it push on tags 👍